src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -113,6 +113,15 @@ class SeekBar extends Slider {
    * @listens mousedown
    */
   handleMouseDown(event) {
+    // only allow left-click seeking
+    if (event.button === undefined && event.buttons === undefined) {
+      // touch screen, not apply, do nothing
+    } else if (event.button === 0 && event.buttons === undefined) {
+      // touch screen, safari on ios
+    } else if (event.button !== 0 || event.buttons !== 1) {
+      return;
+    }
```

Comment:
You could potentially do something like:
```js
  // only allow left-click seeking
if (
  // touch screen, not apply, do nothing
  !(event.button === undefined && event.buttons === undefined) &&
  // touch screen, safari on ios
  !(event.button === 0 && event.buttons === undefined) &&
  (event.button !== 0 || event.buttons !== 1)
) {
 return;
}
```

Also, we probably should pull this out into a tester function because keeping this copied between the two places is bound to cause issues down the line. Maybe moved to dom.js as a `isNotLeftClick(event)` or something like that and can be used something like so:
```js
if (isNotLeftClick(event)) {
  return;
}
```